packages/design-system/src/components/FormLabel/FormLabel.jsx

```diff
@@ -51,6 +36,18 @@ export class FormLabel extends React.PureComponent {
     );
   }
 
+  errorMessage() {
+    if (this.props.errorMessage) {
+      // Include fallback for errorId for usage outside of FieldContainer
+      const errorId = this.props.errorId || `${this.props.fieldId}-error`;
```

Comment:
Normally `errorId` is passed into `FormLabel` via `FormContainer`, but some apps are using `FormLabel` independently.